src/odemis/acq/fastem.py

```diff
@@ -76,3 +77,26 @@ def _run_fake_acquisition():
 
 def estimateTime(roas):
     return len(roas) * 2
+
+
+# Overview acquisition
+def acquire_overview(coords, sem_stream):
+    """
+    :param coords: (float, float, float, float) minx, miny, maxx, maxy coordinates of overview region
+    :param sem_stream (SEMStream):
+    :returns: (ProgressiveFuture): acquisition future
+    """
```

Comment:
Note that my vision of the interface is now `acq.fastem.acquireTiledArea(stream, stage, area, live_stream=None) → Future`. But as this file is going to be changed anyway, I don't think it matters much.